src/backend.py

```diff
@@ -589,34 +588,43 @@ def app_setScoreCap(request):
     json_data = request.data
     if "on-machine" in json_data:
         info = ds_read_record("extras", 0)
-        info["other"] = json_data["on-machine"]
+        info["enter_intials_on_game"] = bool(json_data["on-machine"])        
         ds_write_record("extras", info, 0)
 
 
 @add_route("/api/settings/tournament_mode")
-def app_getTournamentMode(request):
-    import SharedState
-
-    return json_dumps({"tournament_mode": SharedState.tournamentModeOn}), 200
+def app_getTournamentMode(request):    
+    tournament_mode = ds_read_record("extras", 0)["tournament_mode"]
+    return json_dumps({"tournament_mode": tournament_mode}), 200
```

Comment:
```suggestion
    return {"tournament_mode": tournament_mode}
```
This is something I should have caught previously. But in add_route, if you return a dictionary/list it will automatically convert to json and add the 200 if you didn't already.